tools/bench/parsec/evm/contracts/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Load generator for PARSEC EVM",
   "main": "index.js",
   "scripts": {
-    "build": "npx hardhat compile && python3 gen_header.py && clang-format -i cpp_header/contracts.hpp"
+    "build": "npx hardhat compile && source scripts/activate-venv.sh && python gen_header.py && deactivate && clang-format -i cpp_header/contracts.hpp"
```

Comment:
This change breaks the build instructions specified in the Readme within this directory (pnpm tries to run build in `sh` not `bash` and `sh` does not have `source`). We can probably just revert this change, anyone who is using this tool is likely not worried about creating their own python environment with access to the `os`, `json` and `re` packages.

Reply:
ACK - will do